tests/unit/detection/test_detection_model.py

```diff
@@ -69,71 +78,133 @@ def test_detection_learner_init_model(od_detection_dataset):
 
 
 @pytest.mark.gpu
-def test_detection_learner_train_one_epoch(od_detection_learner):
+def test_detection_learner_train_one_epoch(
+    od_detection_learner,
+    od_detection_mask_learner
+):
     """ Simply test that a small training loop works. """
     od_detection_learner.fit(epochs=1)
+    # test mask learner
+    od_detection_mask_learner.fit(epochs=1)
 
 
 @pytest.mark.gpu
-def test_detection_learner_plot_precision_loss_curves(od_detection_learner):
```

Comment:
maybe split this up too. and all the ones below as well.

Reply:
OK